core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-box/template-builder-box.component.ts

```diff
@@ -0,0 +1,33 @@
+import { NgClass, NgFor, NgIf } from '@angular/common';
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+import { ButtonModule } from 'primeng/button';
+import { CardModule } from 'primeng/card';
+import { ScrollPanelModule } from 'primeng/scrollpanel';
+
+export enum TemplateBuilderBoxSize {
+    large = 'large',
+    medium = 'medium',
+    small = 'small'
+}
+
+@Component({
+    selector: 'dotcms-template-builder-box',
+    templateUrl: './template-builder-box.component.html',
+    styleUrls: ['./template-builder-box.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    standalone: true,
+    imports: [NgFor, NgIf, NgClass, ButtonModule, CardModule, ScrollPanelModule]
+})
+export class TemplateBuilderBoxComponent {
+    items = [
+        { label: 'demo.dotcms.com' },
+        { label: 'System Container' },
+        { label: 'demo.dotcms.com' },
+        { label: 'demo.dotcms.com' },
+        { label: 'demo.dotcms.com' },
+        { label: 'demo.dotcms.com' }
+    ];
```

Comment:
We need to discuss if this comes from an input or from where. But whatever we do we can mock it in Storybook.